modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaJAXRSServerCodegen.java

```diff
@@ -281,5 +313,8 @@ public void setUseBeanValidation(boolean useBeanValidation) {
         this.useBeanValidation = useBeanValidation;
     }
 
-
+    @VisibleForTesting
```

Comment:
`VisibleForTesting` is only for testing purposes and should probably be removed in favor of a package-private visibility.